js/views/attachment_view.js

```diff
@@ -15,8 +17,12 @@
 
   var ImageView = Backbone.View.extend({
       tagName: 'img',
-      initialize: function(dataUrl) {
-          this.dataUrl = dataUrl;
+      // TODO: Disable JSHint to allow for `{blob}={}` syntax:
+      initialize: function(blobUrl, options) {
```

Comment:
Would be cool to use `{blob} = {}` destructuring syntax for function arguments, e.g. `options` to make code clearer and shorter but currently this is not parsed correctly by JSHint.